httpclient5-cache/src/main/java/org/apache/hc/client5/http/impl/cache/ResponseCachingPolicy.java

```diff
@@ -50,6 +51,29 @@
 
 class ResponseCachingPolicy {
 
+    /**
+     * The default freshness duration for a cached object, in seconds.
+     *
+     * <p>This constant is used to set the default value for the freshness lifetime of a cached object.
+     * When a new object is added to the cache, it will be assigned this duration if no other duration
+     * is specified.</p>
+     *
+     * <p>By default, this value is set to 300 seconds (5 minutes). Applications can customize this
+     * value as needed.</p>
+     */
+    private static final long DEFAULT_FRESHNESS_DURATION = 300L;
```

Comment:
@arturobernalg This is actually the case where the use of `java.time.Duration` instead of plain `long` might be beneficial.